src/mdesktopentry.cpp

```diff
@@ -575,3 +576,16 @@ bool MDesktopEntry::isSandboxed() const
 {
     return d_ptr->keyFile.hasSection(SailjailSection) && value(SailjailSection, SandboxingKey) != DisabledValue;
 }
+
+QList<MDesktopAction> MDesktopEntry::actions() const
+{
+    QList<MDesktopAction> actionsList;
+    QStringList actions = value(DesktopEntrySection, ActionsKey).split(";", Qt::SkipEmptyParts);
+    if (!actions.empty()) {
```

Comment:
Just cosmetics, doesn't matter much, but this empty check shouldn't provide much when the for loop just exits immediately anyway.